### DISCUSSION TITLE
feature request

### DISCUSSION THREAD

**Discussion by khaled-ib** [osmandapp/OsmAnd#11365] [Ideas]
hello
i really do love your app.
and want to suggest something most really want it specially off-road navigation.
i bought this app for my iphone and also for my car android screen, in the car android screen for map markers, if i want to enter coordinates, will go to map markers then other then enter coordinates, it’s easy but the annoying after entering the coordinates i need to name it, and where to save it then need to search for it again , hope if there is a choice GoTo if i enter the coordinates then it immediately go to it, it’s simple and easy and many will love it and hope it will be added to ios as well.
great job and thanks

**Comment by Ungoose** [comment]
Have you tried entering the coordinates into the search field? You will immediately go to the point you are interested in and can place a marker there if you want (no renaming or saving required) or just not mark it at all. Sounds close to what you want.

**Reply by garrettboone** [reply]
By chance do you know if this action is the addMapMarker in the api? I want to send coordinates from another app and have OsmAnd open to that location on the map, but not have to name it, as you are describing.

**Reply by Lee-Carre** [reply]
@garrettboone

> I want to send coordinates from another app

Use a `geo:` URI intent from the other app. OsmAnd handles those (and you can then tell it to do various things with that marked point).

**Comment by Lee-Carre** [comment]
There's a dedicated coördinates-input UI under the address tab of the `search` UI. That places a temporary marker at that point on the map.

If that's the desired destination, then simply tap the `Navigation` widget in the details pane which pops up, to have OsmAnd calculate a route to it without further fuss.

Of course, you could add it as a marker / favourite, instead, if that makes more sense in other cases. OsmAnd is very flexible.

If these coördinates are coming from another app (rather than paper or a different device), then you could tell the other app to send an `intent`; OsmAnd can handle `geo:` URI intents from other apps for easier importing / transfer.
Importing a GPX is also possible, if available from your data-source.

If you're doing Geocaching (as a guess), there are (yet) other apps, for making that easier.
